docker/Dockerfile.el7

```diff
@@ -0,0 +1,6 @@
+FROM centos:7
+
+COPY docker/assets/jortel-gofer-epel.repo /etc/yum.repos.d/jortel-gofer-epel.repo
```

Comment:
Could this be `container/assets` folder? Or I have seen `container-assets`. There's no need to make this docker specific.

Reply:
I'm cool with that